src/main/kotlin/nl/avisi/structurizr/site/generatr/site/views/AutoReloading.kt

```diff
@@ -0,0 +1,37 @@
+package nl.avisi.structurizr.site.generatr.site.views
+
+import kotlinx.html.*
+import nl.avisi.structurizr.site.generatr.site.asUrlRelativeTo
+import nl.avisi.structurizr.site.generatr.site.model.PageViewModel
+
+fun HEAD.script(viewModel: PageViewModel) {
+    script(
+        type = ScriptType.textJavaScript,
+        src = "../" + "/auto-reload.js".asUrlRelativeTo(viewModel.url)
+    ) { }
+}
+
+fun BODY.progress() {
```

Comment:
Same as the `script` function.

```suggestion
fun BODY.autoReloadProgress() {
```